e2e/basic/suite_setup_test.go

```diff
@@ -0,0 +1,41 @@
+package basic
+
+import (
+	"context"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/celestiaorg/knuu/pkg/knuu"
+)
+
+type Suite struct {
```

Comment:
We should update this to be a unique name from `suite.Suite` to avoid errors from import conflicts. 
```suggestion
type TestSuite struct {
```
This change would need to be reflected in the methods in this file and the test you updated

Reply:
Here we use it as TestSuite as well https://github.com/celestiaorg/knuu/pull/416/files#diff-f45d80f83b805ff61cdd3c3aa3a3de08045cdc644649e042af4ff3a786f78855R16